README.md

```diff
@@ -2,8 +2,20 @@ User config tool for #!
 
 Lets users change ldap details, ssh keys, etc.
 
-TODO. Document
+## Documentation 
 
-Install: `pip install git+https://github.com/hashbang/hashbangctl`
+Typing ``hashbangctl`` when you're connected to one of the hashbang servers will start the programm and give you some details about your user.
+Follow the instructions on the screen to change your 
+
+* username 
+* shell(``cat /etc/shells`` will list all available shells) 
+	* add or delete SSH Keys
```

Comment:
indentation here isn't consistent: it's okay to use tabs, but you still need to use a space before the `*`.
Note that usually we use `  - ` (`<space><space><hyphen><space>`) to create markdown lists. So the indentend form with tabs is: `<tab><space><space><hyphen><space>`